UKHomeOffice/nmsw-ui
# NMSW-1294 Update webpack hash

# Ticket

NMSW-1295

----

## AC

resolve the warning
```
(node:99961) [DEP_WEBPACK_TEMPLATE_PATH_PLUGIN_REPLACE_PATH_VARIABLES_HASH] DeprecationWarning: [hash] is now [fullhash] (also consider using [chunkhash] or [contenthash], see documentation for details)
(Use `node --trace-deprecation ...` to show where the warning was created) 
```

We were using hash instead of contenthash in the MiniCssExtractPlugin config - have updated that now

----

## To test

- `npm run build` should build without the warning
- `npm run start` should look normal in the UI

----

## Notes
